WiFiManager.cpp

```diff
@@ -492,7 +504,21 @@ void WiFiManager::handleWifi(boolean scan) {
       break;
     }
 
-    String pitem = FPSTR(HTTP_FORM_PARAM);
+	String pitem;
+	switch (_params[i]->getLabelPlacement()) {
+    case 1:
```

Comment:
Instead of using numbers, please define 1 and 2 as constants/defs then use the constants/defs.